src/styles/style.css

```diff
@@ -0,0 +1,98 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+body {
+  margin: 0;
+  font-family: roboto, sans-serif;
+}
+
+.card {
+  border: solid 1px #f3f3f3;
+  width: 200px;
+  height: 408px;
+  background-image: url('../images/imac.jpeg');
+  background-position-y: 32px;
+  background-position-x: 19px;
+  background-size: 160px, 134px;
+  background-repeat: no-repeat;
+}
+
+.card__text {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: #060b35;
+  margin-top: 206px;
+  padding: 0 16px;
+}
+
+.card__prod-code {
+  color: #616070;
+  font-size: 10px;
+  padding: 4px 16px;
+}
+
+.card__reviews {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-top: 16px;
+  padding: 0 16px;
+}
+
+.card__reviews-text {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-left: 17px;
+}
+
+.card__price {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+  margin-top: 24px;
+  padding: 0 16px;
+  display: flex;
+  flex-direction: row;
+}
+
+.card__price-value {
+  margin-left: 85px;
+  font-weight: 700;
+  font-size: 16px;
+  color: #000;
+}
+
+.card__button-buy {
+  width: 166px;
+  height: 40px;
+  border-width: 0;
+  border-radius: 5px;
+  background-color: #00acdc;
+  margin: 16px;
+  color: #fff;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+}
+
+.card:hover .card__button-buy {
+  background-color: #fff;
+  color: #00acdc;
+  border: solid, 1px, #00acdc;
```

Comment:
There is a syntax error in the border property. Commas should not be used between values. It should be 'border: solid 1px #00acdc;' instead of 'border: solid, 1px, #00acdc;'.